### PR TITLE
doc: release notes for 14.2.1

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by theanalyst** [comment]
The changelog still needs some work, mainly the notable section needs an entry on RBD jewel incompatiblity, @dillaman  how do you want to word that?

**Comment by dillaman** [comment]
Perhaps just a "Known Issues" section that just says "Nautilus-based librbd clients cannot open images stored on pre-Luminous Ceph clusters"

**Comment by theanalyst** [comment]
jenkins render docs

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -1,3 +1,169 @@
...
+
+* Ceph now packages python bindings for python3.6 instead of
+  python3.4, because EPEL7 recently switched from python3.4 to
+  python3.6 as the native python3. see the `announcement <https://lists.fedoraproject.org/archives/list/epel-announce@lists.fedoraproject.org/message/EGUMKAIMPK2UD5VSHXM53BH2MBDGDWMO/>_`
+  for more details on the background of this change.
+
+Known Issues
+------------
+
+* Nautilus based librbd clients cannot open images stored on pre Luminous
```
Comment (liewegas) [inline doc/releases/nautilus.rst]: `Nautilus-based` and `pre-Luminous`

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by theanalyst** [comment]
fixed a broken link, should be ready now

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
The changelog entries could be sorted?

**Comment by theanalyst** [comment]
> The changelog entries could be sorted?

done

**Pull request merged**